plugin/evm/atomic/export_tx.go

```diff
@@ -307,22 +306,12 @@ func NewExportTx(
 	}}
 
 	var (
-		avaxNeeded           uint64 = 0
+		avaxNeeded           uint64 = amount
```

Comment:
nit let's then remove the variable entirely and rename the amount argument with `avaxNeeded`?